### PR TITLE
Batch write for state trie migration

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes
This PR introduces batch write for state trie migration.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments
I will rebase this PR after previous PR merged.

### CODE REVIEW

**Comment by KimKyungup** [comment]
@aidan-kwon @winnie-byun I rebased. PTAL. :)

File: blockchain/blockchain.go [hunk 1]
```diff
@@ -275,6 +275,22 @@ func (td *stateTrieMigrationDB) ReadPreimage(hash common.Hash) []byte {
 	return td.ReadPreimageFromNew(hash)
 }
 
+func (bc *BlockChain) stateMigrationCommit(s *statedb.TrieSync, db database.DBManager) (int, time.Duration, error) {
+	start := time.Now()
+	stateTrieBatch := db.NewBatch(database.StateTrieDB)
+
+	written, err := s.Commit(stateTrieBatch)
+	if written == 0 || err != nil {
```
Comment (aidan-kwon) [inline blockchain/blockchain.go:283]: Do we need to check `written == 0`?

Comment (KimKyungup) [reply blockchain/blockchain.go:283]: I get this code from present fast sync code. :)
I think this check to be able to prevent call `Write` with no item.

**Comment by KimKyungup** [comment]
@winnie-byun PTAL.

**Pull request merged**